torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -40,6 +40,20 @@ inline Tensor logsigmoid(const Tensor& input) {
   return torch::log_sigmoid(input);
 }
 
+inline Tensor softmax(const Tensor& input, const SoftmaxOptions& options,
+                      c10::optional<torch::Dtype> dtype = c10::nullopt) {
+  int dim = options.dim();
+  Tensor ret;
+
+  if (dtype == torch::Dtype::Undefined) {
```

Comment:
We should change this to
```suggestion
  if (dtype == c10::nullopt) {
```

Reply:
Sorry, I missed this part. Now, it was fixed.